pkg/settings/policy.go

```diff
@@ -43,11 +43,11 @@ func (r *PolicyAgent) Load() (err error) {
 	} else {
 		r.CA = ServiceCAFile
 	}
-	r.Limit.Worker, err = getEnvLimit(PolicyAgentWorkerLimit, 25)
+	r.Limit.Worker, err = getEnvLimit(PolicyAgentWorkerLimit, 10)
 	if err != nil {
 		return err
 	}
-	r.Limit.Backlog, err = getEnvLimit(PolicyAgentBacklogLimit, 250)
+	r.Limit.Backlog, err = getEnvLimit(PolicyAgentBacklogLimit, 1024)
```

Comment:
The backlog has a small memory footprint so this can be bigger.  The number is somewhat arbitrary anyway.